05-28 19:49:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:39 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-28 19:49:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-147]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:202)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:423)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2804)
	at sun.reflect.GeneratedMethodAccessor132.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:123)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Thread.java:662)
05-29 13:09:52 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 13:09:52 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 19:38:55 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 19:38:55 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 23:12:28 jdbc[4]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 23:12:28 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 23:12:28 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 23:15:35 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-29 23:15:35 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 13:15:59 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 13:15:59 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 13:42:22 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 13:42:22 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 13:42:27 jdbc[4]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 14:10:53 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 14:10:53 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 14:18:48 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 14:18:48 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 20:01:44 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 20:01:48 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:08:13 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:08:13 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:38:44 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:38:44 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:46:03 jdbc[4]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:46:03 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
05-31 21:46:03 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 10:49:49 jdbc[4]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 10:49:49 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 10:49:49 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 11:39:24 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 11:39:24 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 13:23:21 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-01 13:23:21 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:04:48 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:04:48 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:07:12 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:07:12 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:27:33 jdbc[3]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.translator.ExecutionFactory.getConnection(ExecutionFactory.java:174)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-02 13:27:33 jdbc[2]: connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:121)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:94)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcXAConnection.<init>(JdbcXAConnection.java:77)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:311)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:222)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:202)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:254)
	at org.teiid.translator.jdbc.JDBCExecutionFactory.getConnection(JDBCExecutionFactory.java:1)
	at org.teiid.translator.ExecutionFactory.getConnection(ExecutionFactory.java:174)
	at org.teiid.dqp.internal.datamgr.ConnectorWorkItem.execute(ConnectorWorkItem.java:199)
	at org.teiid.dqp.internal.process.DataTierTupleSource.getResults(DataTierTupleSource.java:338)
	at org.teiid.dqp.internal.process.DataTierTupleSource.access$0(DataTierTupleSource.java:329)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:138)
	at org.teiid.dqp.internal.process.DataTierTupleSource$1.call(DataTierTupleSource.java:1)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at org.teiid.dqp.internal.process.DQPWorkContext.runInContext(DQPWorkContext.java:197)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$RunnableWrapper.run(ThreadReuseExecutor.java:118)
	at org.teiid.dqp.internal.process.ThreadReuseExecutor$3.run(ThreadReuseExecutor.java:288)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
06-03 11:03:23 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT G_0.COMMONNAME, G_0.UUID, G_0.QUUID, G_0.COMPANYID, G_0.FULLNAME, G_0.CLASSIFICATION FROM COMPANIES AS G_0 WHERE G_0.COMPANYID IN (SELECT G_1.COMPANYID FROM REL_COMP_VEH AS G_1, (SELECT DISTINCT G_2.VEHICLEID AS C_0 FROM VEHICLES AS G_2) AS V_0 WHERE G_1.VEHICLEID = V_0.@[*]VEHICLEID) "; expected "identifier"; SQL statement:
SELECT g_0.COMMONNAME, g_0.UUID, g_0.QUUID, g_0.COMPANYID, g_0.FULLNAME, g_0.CLASSIFICATION FROM COMPANIES AS g_0 WHERE g_0.COMPANYID IN (SELECT g_1.COMPANYID FROM REL_COMP_VEH AS g_1, (SELECT DISTINCT g_2.VEHICLEID AS c_0 FROM VEHICLES AS g_2) AS v_0 WHERE g_1.VEHICLEID = v_0.@vehicleID) [42001-147]
